src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -16,7 +16,18 @@ namespace System.Net.Http.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string Username = "testuser";
```

Comment:
Ack. I will move these two constants to the server constants file. Only point of confusion  was that this pair is not tied to the current test server. The current test server (httpbin.org) can provide a url for any given pair of username and password. In a sense they seemed like test constants than the server constants.